app/containers/ChangeEmail/actions.js

```diff
@@ -107,3 +106,18 @@ export function changeEmailErr(changeEmailError) {
     changeEmailError,
   };
 }
+
+export function getEmailAddress(address) {
```

Comment:
why get? It's set

Reply:
because the function makes a get request and returns data about user's email from back